### ISSUE TITLE
Reproduce V2-S,M and Worse Accuracy

### ISSUE THREAD

**Issue by mwbyeon** [google/automl#1024]
I tried reproduce V2-S,M training on TPUs. (Run main.py with default configurations)
But it was worse accuracy than [pre-trained checkpoints](https://github.com/google/automl/tree/master/efficientnetv2#2-pretrained-efficientnetv2-checkpoints).

## EfficientnetV2-S Evaluation Result
```
Saving dict for global step 107336: 
eval/acc_top1 = 0.83456, 
eval/acc_top5 = 0.96524,
global_step = 107336,
loss = 1.8528475, 
model/flops = 8.411316, 
model/params = 21.459238, 
model/resolution = 384.0
```
I got 83.45%, but pre-trained checkpoint is 83.9%.

## EfficientnetV2-M Evaluation Result
```

Saving dict for global step 107336: 
eval/acc_top1 = 0.8446, 
eval/acc_top5 = 0.96958, 
global_step = 107336, 
loss = 1.854279, 
model/flops = 24.70273, 
model/params = 54.13695, 
model/resolution = 480.0
```
I got 84.46%, but pre-trained checkpoint is 85.2%.

Could you tell me what am I missing?
Thank you.

**Comment by snake-arch** [comment]
Can you please share the concerning code and datasets used?

**Comment by Aakash-here** [comment]
Can you share me the source code?

**Comment by Cocoalate** [comment]
The pretrained model is trained on large dataset so it is robust.When you finetune on the basis of pretrained model you should use very slight learning rate.Check the parameters you set in 'datasets.py'.The author provides an example class named ImagenetFt.
